integration-cli/docker_cli_run_test.go

```diff
@@ -2718,6 +2718,23 @@ func TestRunAllowPortRangeThroughExpose(t *testing.T) {
 	logDone("run - allow port range through --expose flag")
 }
 
+// test docker run expose a invalid port
+func TestRunExposePort(t *testing.T) {
+	runCmd := exec.Command(dockerBinary, "run", "--expose", "80000", "busybox")
+	out, _, err := runCommandWithOutput(runCmd)
+	//expose a invalid port should with a error out
+	if err != nil && !strings.Contains(out, "Invalid range format for --expose") {
+		t.Fatalf("failed to run container, output: %q", out)
+	}
+	if !strings.Contains(out, "Invalid range format for --expose") {
```

Comment:
Failing test would never reach this point since you are testing for this above @ 2726.